examples/RemoteCrashCheck/RemoteCrashCheck.ino

```diff
@@ -28,73 +28,105 @@
 
 #include "EspSaveCrash.h"
 #include <ESP8266WiFi.h>
+#include <ESP8266WiFiMulti.h>
```

Comment:
I would remove this `#include` and all related code below (event if comments provide explanation why the code is there and commented out).
I think that Wi-Fi Multi is not essential to presentation how to check crash information remotely and commented out code clutters example.
Without extra code hidden in comments example will be shorter and easier to analyze to less experienced users.

As result more people are likely to try and use such example :)